doc/content/getting-started/installation/certificates.md

```diff
@@ -17,7 +19,9 @@ $ mkdir ./acme
 $ sudo chown 886:886 ./acme
 ```
 
-Your directory should look like this:
+> `886` is the uid and the gid of the user that runs {{% tts %}} in the Docker container. If you don't set these permissions, you'll get an error saying something like `open /var/lib/acme/acme_account+key<...>: permission denied`.
```

Comment:
Please use `UID`, `GID`. I also think the text should be `you may encounter an error resembling` and then the error text.